ts/helpers/index.ts

```diff
@@ -0,0 +1,5 @@
+import { getAttributeByPath } from "./attributesHandler";
+import { CheckValidObject as checkPath } from "./checkobj";
```

Comment:
@rabi92 how do you feel about renaming this like an accessor? It does return a value by path so perhaps it should be `getObjectAtPath)`

Reply:
yea sounds better, will change

Reply:
Now when i think about it again, we don't necessarily use it to getObject, we also use that as checks to see if the property exists. @shak 

Like in this example below, it makes sense to have it as `checkPath`.
![image](https://user-images.githubusercontent.com/16796013/125194957-8b809980-e264-11eb-9bf6-1f5b7965214f.png)

Reply:
hmm this is the wrong use then. Two solution then

if you are not going to use this as a check often then:
```javascript
if (isEmpty(getObjectAtPath(a, b)) === false) { logic here ... } 
```

OR if it will be used frequently, then add the boolean check as an alias

```javascript
export getObjectAtPath(a, b) { logic here ... };

export checkObjectAtPath(a, b) { return isEmpty(getObjectAtPath(a, b)); };
```

Reply:
can discuss in detail later if not clear

Reply:
isEmpty returns true if the value is a number, so that can be problematic. actually getObjectAtPath will return undefined in a failure case so i can just add a condition to check if it is not undefined. checking at other places, example above and few other places are not checking if its undefined or not. 
![image](https://user-images.githubusercontent.com/16796013/125196995-02ba2b80-e26d-11eb-80c2-e10ef80372fe.png)

Reply:
so i guess `checkObjectAtPath` is fine.

Reply:
@rabi92 I'll discuss this with you

Reply:
pushed the changes as discussed, let me know if you are happy with it. @shak